packages/core/components/Modal/index.tsx

```diff
@@ -14,10 +14,10 @@ export interface ModalProps {
 
 export enum ModalType {
     CodeSnippet = 1,
-    DataSource = 2,
-    MetadataManifest = 3,
-    SmallScreenWarning = 4,
-    CopyFileManifest = 5,
+    CopyFileManifest = 2,
+    DataSource = 3,
+    MetadataManifest = 4,
+    SmallScreenWarning = 5,
```

Comment:
I don't think this is breaking (didn't seem to be for my tests) any thoughts @aswallace?

Reply:
I think it's fine, the only thing might be to also match it in the order of the switch just for consistency. It shouldn't change any functionality though